roles/hxr.admin-tools/tasks/main.yml

```diff
@@ -1,24 +1,12 @@
 ---
 - name: Install useful tools
   yum:
-    name: "{{ admin_packages }}"
+    name: "{{ admin_packages | select | list }}"
     state: installed
 
-- name: Check if firewalld is already installed
-  yum:
-    list: firewalld
-  # If not installed yum_list.results[*].yumstate != installed
-  register: yum_list
-
-- name: Stop firewalld
```

Comment:
fascinating, why would i do that?

I suspect it was because: internally we didn't strictly *need* a firewall, since a lot of that functionality was being provided in duplicate by OpenStack's firewall, and internally we did occasionally need random ports open for new services, but either way, that was some :cowboy_hat_face: nonsense, so, good that you add it back :)